cypress/plugins/index.js

```diff
@@ -15,7 +15,7 @@
 /**
  * @type {Cypress.PluginConfig}
  */
-module.exports = (on, config) => {
+module.exports = (_on, _config) => {
```

Comment:
An example of using the `no-unused-vars` rule with vars we want to keep even if they're not currently in use.